### ISSUE TITLE
Entries without URLs display as options when choosing from entries.

### ISSUE THREAD

**Issue by jsunsawyer** [fruitstudios/LinkIt#73]
I have a couple sections with "Entries in this section have their own URLs" unchecked. These entries display when choosing from entries in a Linkit field.

I know I can manually set which sources entries can be selected from, but it'd be nice if it worked like Craft's rich text links do. When choosing an entry for a link, it won't display entries that don't have links. The Linkit entries field currently behaves more like Craft's entries field.

The problem with manually setting each Linkit field's sources is that if another structure or channel is ever added, I have to go in and update each Linkit field to include the new source.

**Comment by siebird** [comment]
I agree. This can be quite cumbersome when you have a lot of `linkit` fieldtypes and need to go into each fields settings to update the entry sections. An option might be to include all by default and then choose which sections you want excluded. 

In the case above, I could see where some sections with URLs I would want excluded. For example, a confirmation page section where we redirect users after a form submission.
